### PR TITLE
Clarify phrasing of MSYS2 dependencies in README.md.

### PR DESCRIPTION
**Pull request by frewsxcv**
Fixes https://github.com/rust-lang/rust/issues/36115.

### CODE REVIEW

**Comment by frewsxcv** [comment]
cc @barik @petrochenkov @alexcrichton

File: README.md [hunk 1]
```diff
@@ -81,9 +81,10 @@ build.
 
    # Install build tools needed for Rust. If you're building a 32-bit compiler,
    # then replace "x86_64" below with "i686". If you've already got git, python,
-   # or CMake installed and in PATH you can remove them from this list. Note
-   # that it is important that the `python2` and `cmake` packages **not** used.
-   # The build has historically been known to fail with these packages.
+   # or CMake installed and in PATH you can remove them from this list.  Note
+   # that it is important that you do not use the `python2` and `cmake` packages
```
Comment (retep998) [inline README.md]: Why was the bold on "**not**" removed?

File: README.md [hunk 2]
```diff
@@ -81,9 +81,10 @@ build.
...
+   # from the [`msys2` subsystem]. The build has historically been known to fail
```
Comment (petrochenkov) [inline README.md]: The link doesn't seem to work in
```
code
```
block (see the preview).

File: README.md (hunk 1 above)
Comment (frewsxcv) [inline README.md]: Unintentional. Reverted back.

File: README.md (hunk 2 above)
Comment (frewsxcv) [inline README.md]: Ah right, this is a code block. Removed the link.

**Comment by frewsxcv** [comment]
Comments have been addressed.

**Comment by alexcrichton** [comment]
@bors: r+

Thanks!

**Pull request merged**